web/client/plugins/featuregrid/panels/index.jsx

```diff
@@ -35,8 +35,9 @@ const Toolbar = connect(
         disableToolbar: state => state && state.featuregrid && state.featuregrid.disableToolbar,
         disableDownload: state => (resultsSelector(state) || []).length === 0,
         isDownloadOpen: state => state && state.controls && state.controls.wfsdownload && state.controls.wfsdownload.enabled,
+        isSyncActive: state => state && state.controls && state.controls.syncwmsfilter && state.controls.syncwmsfilter.enabled,
         isColumnsOpen: state => state && state.featuregrid && state.featuregrid.tools && state.featuregrid.tools.settings,
-        isSearchAllowed: (state) => !isCesium(state),
+        isSearchAllowed: (state) => true || !isCesium(state),
```

Comment:
this is always true

Reply:
Sorry Mauro, I used this trick to test the sync with Cesium and I forgot to remove!